java/spacewalk-java.spec

```diff
@@ -190,7 +189,7 @@ Requires:       glassfish-jaxb-runtime
 Requires:       glassfish-jaxb-txw2
 Requires:       istack-commons-runtime
 Requires:       (glassfish-jaxb-api or jaxb-api)
-%if 0%{?rhel} == 8
```

Comment:
Same as before. Truth to be told I wonder why this is `== 8` and not `>= 8`. Maybe someone (@sbluhm) noticed that this is already broken for EL9?

Reply:
:) I don't know why I limited it to 8 in the first place. Am wondering whether I thought that there is a kernel fix. Either way, the package is available on 9 and optional. So in best case it solves problems before they show up.

Reply:
Clearly I didn't notice this came from you @sbluhm :-D

In that case, I assume you already kept anything in mind, so removing `>= 8` should be ok, but I guess this one is not, unless you can check that it works with `9`  as well.

Changelog entries should go away, in this case. They are not relevant for customers, and we already have others that will allow me to tag :-)

Reply:
:) It works on 9